src/platform/packages/shared/kbn-unified-doc-viewer/src/components/doc_viewer/doc_viewer.tsx

```diff
@@ -36,16 +36,16 @@ export const DocViewer = forwardRef<DocViewerApi, DocViewerInternalProps>(
   ({ docViews, initialTabId, ...renderProps }, ref) => {
     const tabs = docViews
       .filter(({ enabled }) => enabled) // Filter out disabled doc views
-      .map(({ id, title, render, component }: DocView) => ({
+      .map(({ id, title, component }: DocView) => ({
         id: getFullTabId(id), // `id` value is used to persist the selected tab in localStorage
         name: title,
         content: (
           <DocViewerTab
+            key={id}
             id={id}
             title={title}
             component={component}
             renderProps={renderProps}
-            render={render}
```

Comment:
👏  that we have a solution of all tabs, great work

one thing to consider here, when removing the render function, no custom plugins can register their custom Angular/Vanilla/Strawberry flavored function. when we internally don't use it, there might be some external on prem deployments that make use of it. I haven't heard it for a long time, it's originating from old times, but we can't rule it out. 

Moving forward I wouldn't consider as a breaking change, I don't think it's documented somewhere, however I think it should be highlighted in the PR description, that registering DocViewer tabs with custom render functionality is no longer possible, only React can be used